database/postgres/build.go

```diff
@@ -156,17 +158,25 @@ func (c *client) CreateBuild(b *library.Build) error {
 	// validate the necessary fields are populated
 	err := build.Validate()
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	// send query to the database
-	return c.Postgres.
+	err = c.Postgres.
 		Table(constants.TableBuild).
 		Create(build.Crop()).Error
+
+	if err != nil {
+		return nil, err
+	}
+
+	return build.Crop().ToLibrary(), nil
```

Comment:
Have we confirmed the `build` object has the ID field populated with this change?

We don't explicitly set that ID field in our codebase which is why I ask.

Reply:
When gorm saves the object, it populates the ID field, right?